exporter/elasticsearchexporter/README.md

```diff
@@ -292,10 +292,10 @@ This can be configured through the following settings:
 The Elasticsearch exporter uses the [Elasticsearch Bulk API] for indexing documents.
 The behaviour of this bulk indexing can be configured with the following settings:
 
-- `num_workers` (DEPRECATED, use sending_queue::num_consumers instead): This config is deprecated and ignored. Number of workers publishing bulk requests concurrently.
-- `flush` (DEPRECATED, use sending_queue instead): This config is deprecated and will be ignored. Event bulk indexer buffer flush settings
-  - `bytes`: Write buffer flush size limit before compression. A bulk request will be sent immediately when its buffer exceeds this limit. This value should be much lower than [Elasticsearch's `http.max_content_length`](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html#http-settings) config to avoid HTTP 413 Entity Too Large error. It is recommended to keep this value under 5MB.
-  - `interval`: Write buffer flush time limit.
+- `num_workers` (DEPRECATED, use `sending_queue::num_consumers` instead): This config is deprecated and will be used to configure `sending_queue::num_consumers` if it is not explicitly defined. Number of workers publishing bulk requests concurrently.
+- `flush` (DEPRECATED, use `sending_queue` instead): This config is deprecated and will be used to configure different options for `sending_queue` if they are not explicitly defined. Event bulk indexer buffer flush settings
+  - `bytes` (DEPRECATED, use `sending_queue::batch::max_size` instead): This config is deprecated and will be used to configure `sending_queue::batch::max_size` if not explicitly defined. See the `sending_queue::batch::max_size` for more details.
```

Comment:
Also can we note that the definition of "byte" is different? It was byte in ndjson and after this PR it'll be just an approximation as it is counted in proto size?

Reply:
Added some docs in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/42767/commits/cd1be4f6480640b743d43c3f94bf7f2fe26771d2. Also added a check in `bulkindexer::Add` to flush if the uncompressed size of the bulk indexer exceeds the configured max size as a protection for this case.